tests/Behat/Tests/Testwork/Argument/MixedArgumentOrganiserTest.php

```diff
@@ -1,24 +1,26 @@
 <?php
 
+/** @noinspection PhpUnhandledExceptionInspection */
```

Comment:
I don't think we should add IDE specific tags to the code so I would rather remove this

Reply:
Good spot, I missed that.

Reply:
Since that doesn't affect end users, I'd say it depends mostly on the maintainers/contributors since it would only serve to help development.
In any case, it seems the feeling is unanimous, so out it goes. :)

Reply:
I think it's a "slippery slope" argument. If people are using different tooling, you end up with a codebase full of different IDE-specific comments and eventually they start to create noise/diff conflicts/etc. Or pulls that just upgrade or change comments to suit a new version of an IDE that pollute the history for end-users.

So I agree it's better to just have a policy to stick to standard code & annotations in public projects.